stat/distuv/laplace.go

```diff
@@ -82,13 +82,20 @@ func (l *Laplace) Fit(samples, weights []float64) {
 	// TODO: Rethink quantile type when stat has more options
 	l.Mu = stat.Quantile(0.5, stat.Empirical, sortedSamples, sortedWeights)
 
-	sumWeights := floats.Sum(weights)
-
 	// The scale parameter is the average absolute distance
 	// between the sample and the mean
-	absError := stat.MomentAbout(1, samples, l.Mu, weights)
-
-	l.Scale = absError / sumWeights
+	var absError float64
+	if weights != nil {
```

Comment:
Could you change this to sum up the weights as we pass through? That way we only do one pass through the weights slice.

Reply:
Done